port/webhook/webhookResourceToPortBody.go

```diff
@@ -0,0 +1,114 @@
+package webhook
+
+import (
+	"context"
+
+	"github.com/port-labs/terraform-provider-port-labs/internal/cli"
+)
+
+func webhookResourceToBody(ctx context.Context, state *WebhookModel) (*cli.Webhook, error) {
+	w := &cli.Webhook{
+		Identifier: state.Identifier.ValueString(),
+		Security:   &cli.Security{},
+	}
+
+	if !state.Icon.IsNull() {
```

Comment:
does flex have this functionlality for this direction too?

if not i suggest you add it